test/acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -3,10 +3,13 @@ package meshgateway
 import (
 	"fmt"
 	"testing"
+	"time"
 
 	"github.com/hashicorp/consul-helm/test/acceptance/framework"
 	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
 	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/serf/serf"
```

Comment:
I don't think we need the serf dep. It's being brought in accidentally in namespaces_test.

Reply:
Agree. I thought since we had it then why not, but it looks like we had it accidentally 😄